### PR TITLE
Fix for Facebook's rate limiting

### PR DESCRIPTION
**Pull request by syed-imam**
This one handles the rate limiting of facebook calls

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -3,13 +3,21 @@ import {postFranklyStoriesToFBGraphAPI} from "./main";
 
 dotenv.config();
 
-postFranklyStoriesToFBGraphAPI().then(fbResponse=>{
+postFranklyStoriesToFBGraphAPI().then(facebookPromises=>{
 
-  for(let i = 0 ; i < fbResponse.length; i++)
+  for(let i = 0; i < facebookPromises.length; i++)
```
Comment (kspector) [inline src/index.ts]: @syed-imam what about using `Promise.all` here instead?:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

File: tests/e2e.spec.ts [hunk 2]
```diff
@@ -8,12 +8,18 @@ describe('e2e Tests', () => {
   it('Facebook index Hudson Valley stories', async ()=> {
 
       let promises: any = postFranklyStoriesToFBGraphAPI();
-      await promises.then(r => {
+      await promises.then(async r => {
```
Comment (kspector) [inline tests/e2e.spec.ts]: Same deal here perhaps?

File: src/index.ts (hunk 1 above)
Comment (syed-imam) [reply src/index.ts]: Problem with Promise.all is that it throws an error even if a single promise is failed in the entire list of promises. I want to be able to catch each promise separately, that's the reason.

Comment (kspector) [reply src/index.ts]: @syed-imam got it 👍

**Pull request merged**